specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -2841,6 +3352,13 @@
       "required": true,
       "type": "string",
       "description": "Client Api Version."
+    },
+    "CacheNameParameter": {
+      "name": "cacheName",
```

Comment:
Is it a client level parameter? If not, you need to add `"x-ms-parameter-location": "method"`.

Reply:
Yes it's a client level parameter

Reply:
I think this is a global parameter with scope "x-ms-parameter-location": "method".

Reply:
Correction: it is method, added x-ms-parameter-location extension